OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1224,16 +1234,18 @@ private static int getTimeZoneOffset() {
    }
 
    private static void registerUser() {
-      Log(LOG_LEVEL.DEBUG,
+      logger.debug(
          "registerUser:" +
          "registerForPushFired:" + registerForPushFired +
          ", locationFired: " + locationFired +
          ", remoteParams: " + getRemoteParams() +
          ", appId: " + appId
       );
 
-      if (!registerForPushFired || !locationFired || getRemoteParams() == null || appId == null)
+      if (!registerForPushFired || !locationFired || getRemoteParams() == null || appId == null) {
+         logger.debug("registerUser not possible");
```

Comment:
I think it would be useful to print the values here so we know why registerUser isn't possible since there are 4 possibilities

Reply:
we already had the 
```

logger.debug(
         "registerUser:" +
         "registerForPushFired:" + registerForPushFired +
         ", locationFired: " + locationFired +
         ", remoteParams: " + getRemoteParams() +
         ", appId: " + appId
      );
```

didn't want to repeat the same data